python/adcm/settings.py

```diff
@@ -255,6 +256,11 @@
             'class': 'logging.FileHandler',
             'filename': os.path.join(BASE_DIR, 'data/log/adwp.log'),
         },
+        'stdout_raw': {
```

Comment:
I think the name `stdout` will be enough to understand

Reply:
done